shared/js/media/remote_controls.js

```diff
@@ -368,14 +338,12 @@ MediaRemoteControls.prototype.notifyAppInfo = function(info) {
  * @param {Object} metadata
  */
 MediaRemoteControls.prototype.notifyMetadataChanged = function(metadata) {
+  var onsuccess = function() {};
```

Comment:
Since we expect to `sendMediaMetaData()` successfully as normal without any console log, we will do nothing in  `onsuccess` function.

Therefore, we might can do
`var onsuccess = function() { // To do nothing here, expect onsuccess normally. };`